commands/station.js

```diff
@@ -20,7 +20,12 @@ export const station = async ({ json, experimental }) => {
     console.error('FIL_WALLET_ADDRESS required')
     process.exit(1)
   }
-  if (!FIL_WALLET_ADDRESS.startsWith('f410') && !FIL_WALLET_ADDRESS.startsWith('0x')) {
+  if (FIL_WALLET_ADDRESS.startsWith('f1')) {
+    console.error('Warning: f1... addresses are deprecated')
```

Comment:
A good error message is actionable - it should tell the user how to fix the problem or at least what to do next.

```suggestion
    console.error('Warning: f1... addresses are deprecated and will not receive any rewards.')
    console.error('Please use an address starting with f410 or 0x')
```